openpype/hosts/hiero/api/launchforhiero.py

```diff
@@ -0,0 +1,85 @@
+import logging
+
+from openpype.vendor.python.common.scriptsmenu import scriptsmenu
+from openpype.vendor.python.common.scriptsmenu.vendor.Qt import QtWidgets
```

Comment:
You don't have to do so explicit imports on vendorized modules.
from scriptsmenu import scriptsmenu
from Qt import QtWidgets
```